eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,42 @@
+$Language = "go"
+$LanguageDisplayName = "go"
+
+function Get-AllPackageInfoFromRepo ($serviceDirectory)
+{
+    $allPackageProps = @()
+    $searchPath = Join-Path $RepoRoot "sdk"
+    if ($serviceDirectory)
+    {
+        $searchPath = Join-Path $searchPath $serviceDirectory 
+    }
+
+    $pkgFiles = Get-ChildItem -Path $searchPath -Include "go.mod" -Recurse
+
+    foreach ($pkgFile in $pkgFiles)
+    {
+        $pkgPath = $pkgFile.DirectoryName
+        $pkgName = $pkgFile.Directory.Name
+        $pkgVersion = ""
+        $serviceDirectory = $pkgName
```

Comment:
Some of the packages in this repo have a service directory and a package directory we should try to handle both.